website/modules/@apostrophecms/search/views/index.html

```
@@ -1,68 +1,115 @@
+      {% endfor %}
+    </nav>
     {% else %}
-      <p class="sf-search-page__empty">
-        {{ __('Sorry, there are no results that match your search.') }}
-      </p>
+    <p class="sr-empty">
+      Sorry, there are no results that match your search.
+    </p>
     {% endif %}
 
     {% if data.totalPages > 1 %}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)